javascript-methods/main.js

```diff
@@ -0,0 +1,56 @@
+const a = 1;
+const b = 2;
+const c = 3;
+const maximumValue = Math.max(a, b, c);
+console.log('maximumValue:', maximumValue);
+
+const heroes = ['Batman', 'Iron Man', 'Spiderman', 'Thor'];
+
+let randomNumber = Math.random();
+randomNumber = randomNumber * heroes.length;
+const randomIndex = Math.floor(randomNumber);
+console.log('randomIndex:', randomIndex);
+const randomHero = heroes[randomIndex];
+console.log('randomHero:', randomHero);
+
+const library = [
+  {
+    title: 'Crying in HMart',
+    author: 'Michelle Zauner',
+  },
+  {
+    title: 'Ender"s Game',
```

Comment:
If you still want to use a `'` you can use an escape character so it'd look like this `title: 'Ender\'s Game'`

The `\` prevents the middle single quote from rendering as the end of the string.